plugins/outputs/azure_data_explorer/azure_data_explorer.go

```diff
@@ -34,10 +36,17 @@ type AzureDataExplorer struct {
 	MetricsGrouping string          `toml:"metrics_grouping_type"`
 	TableName       string          `toml:"table_name"`
 	CreateTables    bool            `toml:"create_tables"`
-	client          localClient
-	ingesters       map[string]localIngestor
-	serializer      serializers.Serializer
-	createIngestor  ingestorFactory
+	//Deprecated: client of type *kusto.Client, ingestors of type ingest.Ingestor introduced
+	client    localClient
+	ingesters map[string]localIngestor
+	/***/
+	serializer serializers.Serializer
+	//Deprecated
+	createIngestor ingestorFactory
+	/***/
+	IngestionType   string `toml:"ingestion_type"`
```

Comment:
Can you please move the exported (config option) up to the other config options so they are grouped in exported/non-exported!?